src/similarActionCheck.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @flow
+ */
+import { isEqual } from 'lodash';
```

Comment:
It'd be very useful to cover the new functionality with unit tests, to verify that indeed it works as intended. Would you mind adding them?. Happy to help if needed, just ping me.
I'll try to be more active from now on.